### ISSUE TITLE
Broadcast global message / Push notifications by administrator/console

### ISSUE THREAD

**Issue by mmarif4u** [go-gitea/gitea#2345]
It would be useful to have broadcast message globally in case Admins want the users know about any downtime/updates etc.

**Comment by bkcsoft** [comment]
There has to be an API _and_ a sub-command (`./gitea broadcast -m "deploying in 5 minutes"`) for scriptability 😉

**Comment by appleboy** [comment]
add a new notification to all users? right?

![screen shot 2017-08-24 at 9 54 59 am](https://user-images.githubusercontent.com/21979/29645812-592368b0-88b2-11e7-9686-b1efde17f743.png)

**Comment by bkcsoft** [comment]
@appleboy No, display a large banner for broadcasts. I'm assuming something like GitLab has:
![](http://obv0ef5sf.bkt.clouddn.com/gitlab-admin-message.png)

**Comment by mmarif4u** [comment]
@bkcsoft you are right. Gitlab way is the proper way so far(to me). We are using gitlab for work within our team and its easy to broadcast a message for maintenance etc.

**Comment by lcges** [comment]
Nice idea.
Are you working on it? It is a concept?

**Comment by sapk** [comment]
Just finding this issue. This would also need to be advertise on git login (similar to gitlab).

**Comment by guillep2k** [comment]
If this is implemented with anything but an e-mail it will be difficult for admins to broadcast downtime notices effectively. Perhaps it needs both kinds of notifications (selectable): e-mail and web notice.

I don't mean to add complexity to this issue, but here's a few related ideas:

* Setup welcome messages.
* Notices for an entire organization, or even teams.
* Notices for repo watchers from owners.
